### PR TITLE
Automated comparison with API calls in process_search_calls UAT

### PR DESCRIPTION
**Pull request by gyanitsa-cb**
## Pull request checklist

Please check if your PR fulfils the following requirements:

- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been added that prove the fix is effective or that the feature works.
- [x] New and existing tests pass locally with the changes.
- [x] Code follows the style guidelines of this project (PEP8, clean code, linter).
- [x] A self-review of the code has been done.

## What is the ticket or issue number?

https://jira.carbonblack.local/browse/CBAPI-2565 

## Pull Request Description

Automated comparison with API calls to remove the need of manual steps. 
Enriched events validations removed from the test. See https://github.com/carbonblack/carbon-black-cloud-sdk-python/pull/171 for their new place.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/tests/uat/process_search_calls.py [hunk 1]
```diff
@@ -407,33 +550,31 @@ def main():
...
+    if print_detail:
+        print("Start date: {}\nEnd date: {}".format(start_date, end_date))
+
+    run_process_invalid_search(cb, print_detail)
+    run_process_event_invalid_search(cb, print_detail)
+    get_list_of_available_process_results(cb)
+    process_guid = get_process_guid(cb, print_detail, start_date, end_date)
+    process_name = retrieve_process_name_from_process_details(cb, process_guid)
+    get_process_facet(cb, print_detail, process_name, start_date, end_date)
+    get_process_events_for_single_process(cb, process_guid, start_date, end_date, print_detail)
```
Comment (gyanitsa-cb) [inline src/tests/uat/process_search_calls.py:575]: Currently fails due to https://jira.carbonblack.local/browse/CBAPI-2588. 
To run the test, comment this validation.

**Review by emitreva-cb — APPROVED** [review]
LGTM

File: src/tests/uat/process_search_calls.py [hunk 2]
```diff
@@ -407,33 +544,31 @@ def main():
...
-    window = '-' + args.window
-
-    do_process = True
-    do_enriched_events = True
+    global ORG_KEY
+    global HOSTNAME
 
-    if print_detail:
-        print(f"profile being used is {args.__dict__}")
+    print_detail = False
```
Comment (hkaragitliev-cb) [inline src/tests/uat/process_search_calls.py:550]: It seems the only way to use the print_detail is to change it in the code, which is not optimal. We should either remove it, or just add another parser.argument for it just like the --window one.

**Review by hkaragitliev-cb — APPROVED** [review]
Good job, just one minor issue that is from before your changes.

**Review by avanbrunt-cb — APPROVED** [review]
Awesome job! This was a bug effort to go through

File: src/tests/uat/process_search_calls.py (hunk 2 above)
Comment (gyanitsa-cb) [reply src/tests/uat/process_search_calls.py:550]: Jira ticket created to cover this issue in all UAT tests where applicable: CBAPI-2613

**Pull request merged**